src/mame/drivers/kaneko16.cpp

```diff
@@ -170,40 +180,32 @@ WRITE16_MEMBER(kaneko16_state::kaneko16_soundlatch_w)
 }
 
 /* Two identically mapped YM2149 chips */
-
-READ16_MEMBER(kaneko16_state::kaneko16_ay1_YM2149_r)
+template<int chip>
```

Comment:
Please use `CamelCase` for template parameters to make it clear they aren't locals or function parameters.